src/package.json

```diff
@@ -46,31 +49,35 @@
   "devDependencies": {
     "@babel/core": "^7.12.0",
     "@storybook/addon-knobs": "^6.0.28",
+    "@nuxtjs/eslint-config": "^0.0.1",
+    "@nuxtjs/eslint-module": "^3.0.2",
+    "@nuxtjs/tailwindcss": "^4.2.1",
     "@storybook/addon-actions": "^6.0.28",
     "@storybook/addon-essentials": "^6.0.28",
     "@storybook/addon-links": "^6.0.28",
     "@storybook/vue": "^6.0.28",
     "babel-eslint": "^10.0.1",
     "babel-loader": "^8.1.0",
-    "babel-preset-vue": "^2.0.2",
     "eslint": "^5.14.1",
     "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-config-standard": "^12.0.0",
     "eslint-import-resolver-webpack": "^0.11.0",
     "eslint-loader": "^2.1.2",
     "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-jest": "^22.3.0",
+    "eslint-plugin-node": ">=7.0.1",
     "eslint-plugin-nuxt": "^0.4.2",
+    "eslint-plugin-promise": ">=4.0.1",
+    "eslint-plugin-standard": ">=4.0.0",
     "eslint-plugin-vue": "^5.2.2",
-    "extract-css-chunks-webpack-plugin": "^4.8.0",
-    "node-sass": "^4.11.0",
+    "node-sass": "^4.14.0",
```

Comment:
I think we want >=5 or 6 for better M1 dev exp, I don't dare merging the pr in liker-land yet, maybe you can try it here?
https://github.com/likecoin/liker-land/pull/523